tests/Testcontainers.EventHubs.Tests/EventHubsContainerTest.cs

```diff
@@ -0,0 +1,71 @@
+namespace Testcontainers.EventHubs;
+
+public abstract class EventHubsContainerTest : IAsyncLifetime
+{
+    private readonly AzuriteContainer _azuriteContainer;
+
+    private EventHubsContainer _eventHubsContainer;
+
+    private readonly INetwork _network = new NetworkBuilder().WithName(NetworkName).Build();
+    
+    private const string NetworkName = "eh-network";
+    private const string AzuriteNetworkAlias = "azurite";
+    
+    private const string EventHubName = "testeventhub";
+    private const string EventHubConsumerGroupName = "testconsumergroup";
+    
+    private EventHubsContainerTest()
+    {
+        _azuriteContainer = new AzuriteBuilder()
+            .WithNetwork(_network)
+            .WithNetworkAliases(AzuriteNetworkAlias)
+            .Build();
+    }
```

Comment:
This is an idea still but due to EventHubs emulator will not work without Azurite then we can provide two ways to run EventHub Container implementation.

1. Embed Azurite, so, users don't worry about it.
2. Allow external Azurite in case users need a specific configuration

/cc @kiview WDYT?

Reply:
Does a pattern exist already ?